### ISSUE TITLE
iRobot integration docs silent on Position, Selective Room Cleaning

### ISSUE THREAD

**Issue by phrz** [home-assistant/home-assistant.github.io#20631]
### Feedback

[This page](https://www.home-assistant.io/integrations/roomba/) does not make reference to where, if anywhere, the current position of the Roomba/Braava is exposed, and does not explain whether Selective Room Cleaning is supported, and if so, how to modify the assigned rooms (e.g. in a service call). This would be extremely helpful, as without these instructions, I am left to search the source code for some suggestion.

### URL

https://www.home-assistant.io/integrations/roomba/

### Version

2021.11.5

**Comment by phrz** [comment]
Hello, this issue is still relevant insofar as I'm unsure if any Roomba team members exist to answer it, and it is an important documentation issue.

**Comment by phrz** [comment]
It is still unclear whether the Roomba integration allows for selective room cleaning or positioning.

Do the code owners know anything about this?
@pschmitt @cyr-ius @shenxn

Any of the documentation contributors?
@frenck
@shenxn
@bdraco
@freekode
@meichthys
@scyto
@klaasnicolaas
@ludeeus
@dgomes
@dasevilla
@Cyr-ius
@cschriel
@dmunozv04
@aquesnel

**Comment by klaasnicolaas** [comment]
This kind of action is not appreciated to tag random people just to get an answer and therefore not rewarded with a substantive response about your Roomba issue.

**Comment by phrz** [comment]
Hello Klaas, I apologize for the notification I must have triggered, and I hope it was not a major inconvenience. I am saddened that you find me undeserving of the "reward" of help, but I understand. 

It would certainly make no sense to tag random people, which is why I tagged people who have touched the Roomba documentation. I have been trying to figure this out on my own for several months but to no success. This was a last ditch effort, assuming that I missed someone who actually works with this integration.

I am sorry that I came knocking at your door, it will not be a mistake I make twice.
